portal/static/portal/sass/partials/_images-backgrounds.scss

```diff
@@ -5,6 +5,12 @@ img {
   max-width: 100%;
 }
 
+.flex-images {
```

Comment:
Sounds good :+1: :smile: 
Just for the sake of consistency, can you make it so that this applies to the is-flex class, that way we don't need to create .flex-images as well?